### ISSUE TITLE
Histogram plot of $ADTV for each stock ever traded in portfolio 

### ISSUE THREAD

**Issue by justinlent** [quantopian/pyfolio#135]
Where, $ADTV is Average Daily Trading Volume in $ terms:
 
$ADTV = average( close_price * volume_shares , 90 )

which would be the average $ traded in the security over the last 90-days

This will allow us to do a quick visual analysis of the capacity/liquidity contraints of an algo, and whether it's an algo that could efficiently trade $10k, $100k, $1M, $10M, etc.

**Comment by humdings** [comment]
Would it also be useful to calculate the percentage of daily volume that an algo traded?

**Comment by justinlent** [comment]
@humdings yep definitely.  I think the plot you recommend would need to be the % of ADTV _each_ ticker in the algo on a daily basis, and this would offer a great deal of insight into the the capacity constraints of the algo - but if the algo holds say 100 tickers per day it might be a tough plot to visualize. Thoughts?

**Comment by humdings** [comment]
It might be easier to just report a few tickers that traded the highest percentage of their average daily volume. For the plot I think an average for the whole portfolio would suffice. This sounds like a pretty useful thing to have.

**Comment by justinlent** [comment]
+1 @humdings Showing just like a Top 10 List of the stocks traded by the algo based on their %ADTV sounds great.  I think we can create 2 or 3 separate plots from the ideas generated in this thread and call the whole section of plots something like "Capacity Analysis"

**Comment by twiecki** [comment]
@humdings Any progress on this? Seems like the last item for 0.2 although we can probably also go ahead without it.

**Comment by humdings** [comment]
No, I've been too slammed with other things lately, it should slow down a bit for me after next week though.

**Comment by twiecki** [comment]
OK, no worries.
